Makefile

```diff
@@ -147,7 +146,7 @@ GINKGO_VERSION ?= 2.5.0 # match our go.mod
 GINKGO_ENV ?= GOLANG_PROTOBUF_REGISTRATION_CONFLICT=ignore ACK_GINKGO_DEPRECATIONS=$(GINKGO_VERSION)
 GINKGO_FLAGS ?= -v -tags=purego -compilers=4 --randomize-all --trace -progress -race
 GINKGO_REPORT_FLAGS ?= --json-report=test-report.json --junit-report=junit.xml -output-dir=$(OUTPUT_DIR)
-GINKGO_COVERAGE_FLAGS ?= --cover --covermode=count --coverprofile=coverage.cov
+GINKGO_COVERAGE_FLAGS ?= --cover --covermode=atomic --coverprofile=coverage.cov
```

Comment:
Why `atomic` rather than `count`?

From [ginkgo docs](https://onsi.github.io/ginkgo/#computing-coverage):
>You can also specify the --covermode to be one of set ("was this code called at all?"), count (how many times was it called?) and atomic (same as count, but threadsafe and expensive). If you run ginkgo --race --cover the --covermode is automatically set to atomic.

Seems like `atomic` should only be needed with `--race`

Reply:
I see now that we do have `-race` specified 👍 🤦